src/azul/__init__.py

```diff
@@ -1071,6 +1071,14 @@ def reindex_sources(self) -> list[str]:
     def google_oauth2_client_id(self) -> Optional[str]:
         return os.environ.get('AZUL_GOOGLE_OAUTH2_CLIENT_ID')
 
+    @property
+    def cloudtrail_s3_bucket_region(self) -> Optional[str]:
```

Comment:
Since we have a default 

```suggestion
    def cloudtrail_s3_bucket_region(self) -> str:
```

use `[]` to get the value and so on.